### DISCUSSION TITLE
Iobroker-Fehler 	Init not possible, going to stop: Unknown XML-RPC tag 'TITLE'

### DISCUSSION THREAD

**Discussion by oradke** [OpenCCU/OpenCCU#1896] [🇩🇪 Fragen&Antworten] [answered]
Seit dem letzten Update meiner RaspimaticVM kriegt mein Iobroker keine Verbindung mehr zum Homematic-IP. Im Log steht:

```
Init not possible, going to stop: Unknown XML-RPC tag 'TITLE'
Cannot call init: [http://192.168.188.103:2010, ""] Unknown XML-RPC tag 'TITLE'

```
Habe Firewall mal komplett auf "Vollzugriff" gestellt, kein Unterschied. Meine Raspimatic (Aktuelle Firmwareversion: 3.65.6.20220723
läuft als VM auf einem Synology NAS, der IObroker in Docker auf der selben Synology. Hat bisher auch super funktioniert - bis zum Update.

Hat jemand eine Idee?

Oliver

**Comment by oradke** [comment]
Wenn ich den Port 2010 mit einem Browser direkt ansteuere, dann kriege ich eine fast leere Website mit:
`
${ccuNotReadyHint}`

**Reply by jens-maus** [reply]
Das deutet dann daraufhin das wohl der `HMIPServer` Dienst nicht richtig läuft bzw. verfügbar ist. Ein Blick in die logfiles unter RaspberryMatic bzw. in die WebUI sollte das aufzeigen.

**Reply by oradke** [reply]
Hui... Das Log ist voller Fehlermeldungen, u.a.:
```
Jul 28 09:09:39 RaspiMaticVM daemon.warn cuxd[745]: process_rpc_request(192.168.188.103) - illegal XMLRPC(init) request
Jul 28 09:09:39 RaspiMaticVM daemon.warn cuxd[745]: process_rpc_request(192.168.188.103) - illegal XMLRPC(listDevices) request
...
Jul 28 09:10:08 RaspiMaticVM local0.warn ReGaHss: WARNING: XMLRPC 'getValue': empty interface URL (interface: 1011, params: {"000313C999EA71:1","ACTIVE_PROFILE"}) [CallXmlrpcMethod():iseXmlRpc.cpp:2618]
Jul 28 09:10:08 RaspiMaticVM local0.err ReGaHss: ERROR: XMLRPC 'getValue' call failed (interface: 1011, params: {"000313C999EA71:1","ACTIVE_PROFILE"}) [CallGetValue():iseXmlRpc.cpp:1435]
Jul 28 09:10:08 RaspiMaticVM local0.err ReGaHss: ERROR: CallGetValue failed; sVal = 0 [ReadValue():iseDOMdpHSS.cpp:124]
...
2022-07-28 09:10:16,527 io.vertx.core.impl.BlockedThreadChecker WARN  [vertx-blocked-thread-checker] Thread Thread[vert.x-eventloop-thread-2,5,main] has been blocked for 5156 ms, time limit is 2000 ms 
io.vertx.core.VertxException: Thread blocked
...
```
Hmm... Hilft mir nicht weiter. Sag Dir das was?

**Reply by oradke** [reply]
Achja... Meine Mediola-Oberfläche zeigt auch keine Daten für HM-IP-Thermostate - scheint also wirklich an der Respimatic zu liegen, nicht am IObroker.

**Reply by jens-maus** [reply]
So einfach ist das nicht. Gerade die "blocked-thread" hinweise zeigen doch das irgendetwas wohl deine CCU/RaspberryMatic mit Anfragen bombardiert und deshalb der `HMIPServer` wohl "dicht" macht und deshalb ioBroker dann irgendwann diese Fehlermeldungen zeigt. Also mal langsam und schrittweise vorgehen und mal alle externen Dinge die die CCU/RaspberryMatic nutzen könnten abschalten/runterfahren und dann schauen was passiert wenn du eins nach dem anderen sich wieder damit verbinden lässt.

**Comment by oradke** [comment, marked as answer]
OK, Fehler gefunden. Nachdem ich alles nochmal überprüft hatte stellte sich heraus, daß die Synology VM den USB-Stick nicht verbunden hatte. HM (also NICHT HM-IP) ging, weil ich ein LAN-Gateway habe.

Wie auch immer: USB-Stick in der VM-Konfig aktriviert, Raspi neu gestartet - Fehler weg.

Hoffe, das hilft anderen Usern die den gleichen Fehler haben.
